src/Nox.Solution/Extensions/ObjectExtensions.cs

```diff
@@ -266,4 +268,13 @@ private static string GetIndentation(int level)
     {
         return new string(' ', level * 4);
     }
+
+    private static bool IsReservedKeyword(string word)
```

Comment:
Why do we need this?

Reply:
To fix the code generation of the Enum values with the reserved keywords.